share/setup.nsi.in

```diff
@@ -154,6 +154,19 @@ no_smgroup:
     Pop $R0
 SectionEnd
 
+Section "Bootstrap" SecBootstrap
+    # Download for our bootstrap
+    NScurl::http GET "https://explorer.decenomy.net/bootstraps/SAPP/bootstrap.zip" "$APPDATA\__Decenomy__\bootstrap.zip" /POPUP /END
```

Comment:
where there is SAPP it should be __DSW__

Reply:
@Luke-Larsen if you wont update this pull request and fix what @pedro-at-decenomy mentioned, I will open a new pull request with the correction pointed out by Pedro.
Are you still interested in this pull request and will you update it Luke?